sanity.json

```diff
@@ -1,84 +1,84 @@
 {
-    "Timestamp": "2018-07-28T17:10:29.733Z",
+    "Timestamp": "2019-02-07T17:10:29.733Z",
     "Name": "Sanity",
-    "Version": "08-2018",
-    "License": "Commercial",
+    "Version": "02-2019",
+    "License": "MIT/Commercial",
     "Inception": "05-2016",
-    "Category": "Essential - Professional",
-    "Open Source": "No",
+    "Category": "Essential - Professional - Enterprise",
+    "Open Source": "Yes",
     "Cloud Service": "Yes",
     "On Premises Installation": "No",
-    "Cloud Service Hosted in Europe": "?",
+    "Cloud Service Hosted in Europe": "Yes",
     "Commercial Support Available?": "Yes",
-    "GraphQL API": "No",
+    "GraphQL API": "Yes",
     "GraphQL [Mutations]": "No",
     "GraphQL [Subscriptions]": "No",
-    "REST API": "Yes",
+    "REST API": "Yes, GROQ",
     "REST [Create]": "Yes",
     "REST [Read]": "Yes",
     "REST [Update]": "Yes",
     "REST [Delete]": "Yes",
-    "REST [Upload]": "No",
+    "REST [Upload]": "Yes",
     "Search API": "Yes",
-    "Search Features [Full Text Search]": "No",
+    "Search Features [Full Text Search]": "Yes",
     "Search Features [Stemming]": "No",
     "Search Features [Stop Words]": "No",
     "Search Features [Boosting]": "No",
-    "Search Features [Autocompletion]": "No",
-    "Search Features [Autosuggestion]": "No",
-    "Search Features [Result Highlighting]": "No",
+    "Search Features [Autocompletion]": "NA",
+    "Search Features [Autosuggestion]": "NA",
+    "Search Features [Result Highlighting]": "NA",
     "Search Features [Geospatial Search]": "No",
     "Search Features [Search within uploads (pdf,doc)]": "No",
     "Image Manipulation": "Yes",
-    "Image Focalpoint support": "",
-    "Image Facedetection": "",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "No",
     "Optimized Image Encoding": "Yes",
-    "Asset fingerprinting": "",
+    "Asset fingerprinting": "No",
     "CDN Support": "Yes",
     "Antivirus Scanning": "No",
-    "Custom Binary Handler": "No",
+    "Custom Binary Handler": "Yes",
     "System Requirements": "NA",
     "Supported Databases": "NA",
-    "Clustering": "No",
+    "Clustering": "NA",
     "Docker Support": "NA",
-    "Backup Feature": "No",
+    "Backup Feature": "Yes",
     "Import/Export": "Yes",
     "CLI": "Yes",
     "SDK [Java]": "No",
-    "SDK [C#]": "No",
+    "SDK [C#]": "Yes",
     "SDK [PHP]": "Yes",
     "SDK [JavaScript]": "Yes",
-    "SDK [React]": "No",
+    "SDK [React]": "Yes",
     "SDK [AngularJS]": "No",
-    "SDK [TypeScript]": "No",
+    "SDK [TypeScript]": "Yes",
     "Web Hooks": "Yes",
     "Eventbus": "Yes",
     "Bulk Import": "Yes",
-    "Client Side Forms": "No",
+    "Client Side Forms": "Yes",
```

Comment:
Many ways to do this. This is one of them. https://github.com/sanity-io/netlify-form-sanity